### PR TITLE
Initial commit to update FTS pre-release details

### PR DESCRIPTION
**Pull request by gkhanna79**
This PR updates the SharedFX generated to be 1.1.0-preview1*, in addition to respective installers to have the updated version.

Once CoreCLR and CoreFX 1.1.0-preview1* packages are published, I will do the following:

1) Update M.N.App project.json to reference them
2) Update the JIT and CoreCLR package versions used to R2R compile the SharedFW

@weshaggard @eerhardt @dagood PTAL

CC @leecow @schellap @wtgodbe

### CODE REVIEW

**Comment by eerhardt** [comment]
Do we have Jenkins set up for this branch?

**Comment by eerhardt** [comment]
The changes LGTM, assuming CI passes.

**Comment by gkhanna79** [comment]
@mmitche just merged my PR for the same (https://github.com/dotnet/dotnet-ci/pull/457).

**Comment by gkhanna79** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@gkhanna79 It will take between 1-20 minutes for the initial test this to be valid.  The generation is on a 20 minute cycle (looking for changes in the groovy files)

**Comment by gkhanna79** [comment]
@dotnet-bot test this please

**Comment by mmitche** [comment]
@dotnet-bot test WIndows_NT x64 Release Build
@dotnet-bot test WIndows_NT x64 Debug Build

**Comment by mmitche** [comment]
@dotnet-bot test WIndows_NT x86 Debug Build

File: branchinfo.txt [hunk 1]
```diff
@@ -4,6 +4,6 @@
 MAJOR_VERSION=1
 MINOR_VERSION=1
 PATCH_VERSION=0
-RELEASE_SUFFIX=beta
-CHANNEL=master
-BRANCH_NAME=master
+RELEASE_SUFFIX=preview1
+CHANNEL=preview
+BRANCH_NAME=release/1.1.0
```
Comment (dagood) [inline branchinfo.txt]: :+1: `BRANCH_NAME` is used for auto-update.

Comment (gkhanna79) [inline branchinfo.txt]: @dagood I take it that was FYI for me, right? Or, do you want me to correct something?

Comment (dagood) [inline branchinfo.txt]: @gkhanna79 Yes, just a FYI that auto-update depends on this config change and that it looks good. (I don't think I have have enough core-setup experience to review the rest of the PR well.)

**Comment by gkhanna79** [comment]
@mmitche Can you please restart the Windows jobs when Jenkins restart is complete?

**Comment by mmitche** [comment]
@dotnet-bot test Windows_NT x64 Release Build
@dotnet-bot test Windows_NT x86 Debug Build

**Pull request merged**